.github/ISSUE_TEMPLATE/1.bug.yml

```
@@ -9,7 +9,7 @@ body:
     attributes:
       label: Search keywords
       description: |
-        Your issue may have already been reported! First search for duplicates among the [existing issues](https://github.com/mui/mui-x/issues).
+        Your issue may have already been reported! First search for duplicates among the [existing issues](https://github.com/mui/mui-x/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aclosed).
```

Not sure about this utf8 param

```suggestion
        Your issue may have already been reported! First search for duplicates among the [existing issues](https://github.com/mui/mui-x/issues?q=is%3Aopen+is%3Aclosed).
```

---

Its the same as in the docs. Not sure what it does though and why its there to begin with!

---

I guess we should also remove it from the rest of the links.